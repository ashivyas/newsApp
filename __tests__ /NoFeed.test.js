import React from "react";
import NoFeed from "../src/component/NoFeed";
import renderer from "react-test-renderer";

it('renders correctly', () => {
  const tree = renderer.create(
    <NoFeed />
    ).toJSON();
  expect(tree).toMatchSnapshot();
});
