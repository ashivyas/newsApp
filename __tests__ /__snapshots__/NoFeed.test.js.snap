// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  style={
    Object {
      "marginBottom": 40,
    }
  }
>
  <Text
    allowFontScaling={false}
    style={
      Array [
        Object {
          "color": undefined,
          "fontSize": 12,
        },
        Array [
          Object {
            "color": "#000",
            "fontSize": 30,
          },
          Object {
            "color": "#000",
            "fontSize": 30,
          },
          Object {
            "alignSelf": "center",
            "color": "#F00",
            "fontSize": 100,
            "marginBottom": 20,
          },
        ],
        Object {
          "fontFamily": "Ionicons",
          "fontStyle": "normal",
          "fontWeight": "normal",
        },
      ]
    }
  >
    ïŒ¿
  </Text>
  <View
    style={
      Object {
        "marginBottom": 20,
      }
    }
  >
    <Text
      style={
        Array [
          Object {
            "color": "#000",
            "fontFamily": "System",
            "fontSize": 16,
          },
          Object {
            "alignSelf": "center",
            "color": "#3f3f3f",
            "fontSize": 16,
            "marginBottom": 5,
          },
        ]
      }
      uppercase={false}
    />
  </View>
</View>
`;
